### PR TITLE
Update the README with a presentation of the project

### PR DESCRIPTION
**Pull request by CoCoSol007**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1 +1,27 @@
...
+    <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/rust/rust-original.svg" height="42" width="52" alt="Rust logo">
+    <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/docker/docker-original.svg" height="42" width="52" alt="Docker logo">
+    <img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/vscode/vscode-original.svg" height="42" width="52" alt="VSCode logo">
+    <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/git/git-original.svg" height="42" width="52" alt="Git logo">
+</div>
+
+
+### Repositories
+
+- [`solipr`](https://github.com/solipr/solipr): The Solipr VCS repository.
```
Comment (tipragot) [inline profile/README.md:22]: Why not solipr/solipr?

File: profile/README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+
+<p align="center">
+  <strong>Solipr is a decentralized version control system designed for peer-to-peer collaboration.</strong>
+</p>
+
+<div align="center">
+    <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/rust/rust-original.svg" height="42" width="52" alt="Rust logo">
+    <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/docker/docker-original.svg" height="42" width="52" alt="Docker logo">
+    <img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/vscode/vscode-original.svg" height="42" width="52" alt="VSCode logo">
+    <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/git/git-original.svg" height="42" width="52" alt="Git logo">
```
Comment (tipragot) [inline profile/README.md:16]: Shouldn't they be links?

File: profile/README.md (hunk 1 above)
Comment (CoCoSol007) [reply profile/README.md:22]: It doesn’t look very good if I do so, but if you wish, I can do it.

File: profile/README.md (hunk 2 above)
Comment (CoCoSol007) [reply profile/README.md:16]: Links to what?

**Comment by tipragot** [comment]
Why do you add repositories that does not exists? I think we should add them after making them.

**Review by tipragot — APPROVED** [review]

**Pull request merged**